src/core/devutils/debughandlers/resizedebughandler/ResizeDebugHandler.ts

```diff
@@ -0,0 +1,37 @@
+import { DebugHandler } from "../DebugHandler";
+import ResizeDebugConfig from "./ResizeDebugConfig";
+import { Dimension } from "../../../..";
+import DebugConfig from "../DebugConfig";
+
+export default class ResizeDebugHandler implements DebugHandler {
```

Comment:
This approach doesn't seems to be scalable. If in future we want to enable other functionality we won't be able to do. `ResizeDebugHandler` by name should only handle resizing issue. Eg: If we want debugging for layout or reusing of view we will add API in "DebugHandler" but we can't modify `ResizeDebugHandler`, as it should be handling those.

So, better will be segregate this. Lets meet and discuss more on this.

Reply:
Ideally there could be one hook with all callbacks and different utils which can infer data from them